.github/workflows/ci.yml

```diff
@@ -124,6 +127,7 @@ jobs:
   release-wheel:
     runs-on: 'ubuntu-latest'
     needs: build-wheel
+    if: github.ref_name == 'main'
```

Comment:
To remove before merging. I generally use this job to get wheels for PRs.

Reply:
You can get these at the bottom of a job as well, the only reason to have a release is to make the artifacts permanent: 
![Artifacts](https://github.com/user-attachments/assets/a55a3262-c4c6-4ea4-8b6b-4530c3f6f735)

Reply:
When I worked on the whole wheel build setup I found it more convenient/efficient to use release artifacts, rather than job artifacts.

Reply:
I've reverted this in https://github.com/finch-tensor/Finch-mlir/pull/16/commits/2f0e7977ab0d9da21000af24aa3ee29acbfc03f3.